apps/server/lib/plugins.ts

```diff
@@ -11,24 +10,23 @@ import { GitHubPlugin } from '@balena/jellyfish-plugin-github';
 import { OutreachPlugin } from '@balena/jellyfish-plugin-outreach';
 import { ProductOsPlugin } from '@balena/jellyfish-plugin-product-os';
 import { TypeformPlugin } from '@balena/jellyfish-plugin-typeform';
+import { PluginManager } from '@balena/jellyfish-worker/build/plugin';
```

Comment:
No deep imports (`jellyfish-types` is an exception). `jellyfish-worker` already exports `PluginManager` from the root